extension-manifest-v3/src/store/options.js

```diff
@@ -167,19 +167,19 @@ async function migrateFromMV2() {
 }
 
 export async function observe(property, fn) {
```

Comment:
This is a copy-paste from the https://github.com/ghostery/ghostery-extension/pull/1161/files#diff-ab787eda050f8d097068bea78c9043e942c4480fdb99acf1f1c0213b0bbd859dL169